js/packages/offline-renderer/package.json

```diff
@@ -25,7 +25,7 @@
   },
   "jest": {
     "transformIgnorePatterns": [
-      "node_modules/(?!@elemaudio/core)"
+      "node_modules/(?!@elemaudio/core|emittery)"
```

Comment:
Can we revert this one now that we're not using emittery?

Reply:
Yep! Removed it 👌